homeassistant/components/mobile_app/__init__.py

```diff
@@ -52,6 +53,8 @@
         except ValueError:
             pass
 
+    discovery.load_platform(hass, 'notify', DOMAIN, {}, config)
```

Comment:
We shouldn't do this, we should forward config entry.

Reply:
I don't think we can forward config entries to notify yet?

Reply:
lame

Reply:
Use the async version of `load_platform`, `async_load_platform`, since we're in async context. And create a task to not lock up.